srcpkgs/radicale3/INSTALL.msg

```diff
@@ -0,0 +1 @@
+You are installing the 3.x version of radicale.
```

Comment:
Not sure what the point of this message is, would suggest getting rid of it.

Reply:
It was copied from the radicale 2 template :D

Reply:
Also, radicale3 does make some backwads incompatible changes from radicale2, otherwise I wouldn't be packaging it here. So both probably need to be kept.